feature-utils/poly-look/src/react-components/cards/cardList.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { INITIAL_HISTORY_STATE } from "../../constants/state";
+
+import "./cardList.css";
+
+/**
+ * A list of Cards
+ *
+ * @param {jsx} children Jsx children (Cards)
+ * @param {String} className CSS classes added to the main div
+ * @returns jsx
+ */
+export const CardList = ({ children, className }) => {
+  return <div className={`card-list ${className}`}>{children}</div>;
+};
+
+/**
+ * Card component that makes up a CardList
+ *
+ * @param {jsx} children HTML elements displayed inside the card
+ * @param {Object} navigation Navigation object, if passed will show a button and will trigger actions
+ * @param {History} navigation.history a useHistory generated history (also requires route)
+ * @param {String} navigation.route a router route (also requires history)
+ * @param {Object} navigation.stateChange a router history stateChange (also requires history)
+ * @param {Callback} navigation.onClick onClick function
+ * @param {String} navigation.buttonText the buttons displayed text
+ * @returns jsx
+ */
+export const Card = ({ children, navigation }) => {
+  if (!navigation) return <div className="card">{children}</div>;
```

Comment:
If you think this is in scope, it would probably be better to create a hierarchy out of this. There are too many `if`s, with optional parameters.

Reply:
Essentially this is either a `Card` or a `NavigableCard`, right?